src/components/SQForm/SQForm.tsx

```diff
@@ -46,6 +50,8 @@ function SQForm<Values extends FormikValues>({
   muiGridProps = {},
   onSubmit,
   validationSchema,
+  validateOnBlur,
+  validateOnChange,
```

Comment:
I think both of these props should be `true` by default so this PR does not introduce any breaking changes.

Reply:
Yeah they are already set to true at the Formik level, so this doesn't contain any breaking changes.  If we want to add a belt to our suspenders code wise to for readability and an extra layer of reinforcement then I'm ok with that.  Let me know!

Reply:
Absolutely, thanks for asking!